### PR TITLE
Fix geospatial docs. Fix ST_EnvelopeAsPts and use array(...) syntax.

### PR DESCRIPTION
**Pull request by jwass**
Returns array<Geometry>.

### CODE REVIEW

**Comment by jwass** [comment]
@mbasmanova

File: presto-docs/src/main/sphinx/functions/geospatial.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ Operations
 
     Returns the bounding rectangular polygon of a geometry.
 
-.. function:: ST_EnvelopeAsPts(Geometry) -> Geometry
+.. function:: ST_EnvelopeAsPts(Geometry) -> array<Geometry>
```
[the remark this answers is not in the document]
Comment (jwass) [reply presto-docs/src/main/sphinx/functions/geospatial.rst]: Thanks, @electrum - I didn't realize that.
I added a separate commit to this PR to update to the `array()` syntax for the rest of the file too. I can squash them into one commit - or undo the other ones if you'd prefer.

**Comment by mbasmanova** [comment]
@jwass Jacob, the 1st commit says `Returns array<Geometry>.` Shouldn't it say `Returns array(Geometry)?`

**Comment by jwass** [comment]
@mbasmanova Yup! Missed that. Fixed.

**Pull request merged**